app/admin/requests/Allergen.php

```diff
@@ -22,6 +22,7 @@ public function rules()
             'description' => 'min:2',
             'status' => 'boolean',
             'locations.*' => 'integer',
+            'locations' => 'required',
```

Comment:
For the sake of consistency, the `locations` rule should come before `locations.*`

Reply:
same for the other files